dvc/fs/base.py

```diff
@@ -182,7 +182,10 @@ def info(self, path_info):
         raise NotImplementedError
 
     def getsize(self, path_info):
-        return self.info(path_info).get("size")
+        size = self.info(path_info).get("size")
+        if size is not None:
+            size = int(size)
+        return size
```

Comment:
Was gdrive returning string size? `fs.info` should always return size as int, so prob worth fixing in `gdrive.py` itself.

Reply:
Yeah, there were 2 issues

- after the transfer change we were verifying the destination ODB on both push and pull, when previously we only verified on pull
- the verify on push caused us to read sizes from the gdrive fs, which made the issue w/gdrive returning string sizes show up

I'll move the conversion into our gdrive fs, still waiting for the branch build to finish to make sure there's nothing else

Reply:
@pmrowla Nice catch! :fire: